lib/Coffer/Path.hs

```diff
@@ -128,6 +137,13 @@ newtype EntryPath = EntryPath { unEntryPath :: NonEmpty PathSegment }
 instance A.ToJSON EntryPath where
   toJSON = String . pretty
 
+instance ToSchema EntryPath where
```

Comment:
Hm this instance doesn't look right.

It's generating this schema:

```json
            "EntryPath": {
                "type": "object",
                "required": [
                    "unEntryPath"
                ],
                "pattern": "(/${pathSegment})+",
                "properties": {
                    "unEntryPath": {
                        "items": {
                            "$ref": "#/components/schemas/PathSegment"
                        },
                        "type": "array",
                        "minItems": 1
                    }
                }
            },
```

And `EntryPath` is not a json object, it's a json string. And indeed, the examples shown in the swagger editor are wrong as well:

![image](https://user-images.githubusercontent.com/718166/172159743-c06ddd2e-fd78-46ad-be5a-662c93c7bc17.png)

Reply:
Note: as a result of fixing this, I don't think the instance `ToSchema PathSegment` will be needed anymore.

Reply:
Fixed the `ToSchema EntryPath`. But `ToSchema PathSegment` is required by `ToSchema Directory`